RandomizerMod3.0/MenuChanger.cs

```diff
@@ -401,6 +401,9 @@ void UpdatePoolPreset(RandoMenuItem<string> item)
                 switch (item.CurrentSelection)
                 {
                     case "Mini Super Junk Pit":
+                        DuplicateBtn.SetSelection(true);
+                        HandleProgressionLock();
```

Comment:
Calling `HandleProgressionLock` also sets the rando buttons appropriately, it feels like a redundant double set for these.
I'm not sure where else the function is called, but we could cut off some of the code in these switch-cases knowing how `HandleProgressionLock` behaves